pkg/controller/core/queuedworkload_controller.go

```diff
@@ -75,6 +80,11 @@ func (r *QueuedWorkloadReconciler) Create(e event.CreateEvent) bool {
 	if wl.Spec.Admission == nil {
 		if !r.queues.AddOrUpdateWorkload(wl.DeepCopy()) {
```

Comment:
update the function to return err message that indicates what exactly is missing (could be Queue or ClusterQueue) and use the err.Error() as the message in the condition, we can then remove the curated message you have now.